MAUI/Popup/popup-positioning.md

```diff
@@ -0,0 +1,267 @@
+
+To open the `SfPopup` relative to a view, use the `SfPopup.ShowRelativeToView(View, RelativePosition)` method.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<ContentPage.Content>       
+  <StackLayout VerticalOptions="Start" HorizontalOptions="Center" Padding="20">
+    <Button x:Name="clickToShowPopup" Text="ClickToShowPopup"
+            VerticalOptions="Start" HorizontalOptions="Start"
+            Clicked="ClickToShowPopup_Clicked" />   
+    <sfPopup:SfPopup x:Name="sfPopup">
+    </sfPopup:SfPopup>
+  </StackLayout>
+</ContentPage.Content>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="4" %}
+private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+{
+  // Shows SfPopup at the bottom of the button.
+  sfPopup.ShowRelativeToView(clickToShowPopup, PopupRelativePosition.AlignBottom);
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Displaying a .NET MAUI Popup at Relative position.](Images/popup-positioning/maui-popup-relative-positioning.png)
+
+### Display popup relatively to a view with absolute coordinates
+
+The `SfPopup` can be displayed at an absolute x, y coordinate from the relative position of the specified view by using the following method.
+
+To open the `SfPopup` in the specific x, y coordinate relative to a view, use the `SfPopup.ShowRelativeToView(View, RelativePosition, x-position, y-position)` method.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<ContentPage.Content>       
+  <StackLayout VerticalOptions="Start" HorizontalOptions="Start" Padding="20">
+    <Button x:Name="clickToShowPopup" Text="ClickToShowPopup"
+            VerticalOptions="Center" HorizontalOptions="Start"
+            Clicked="ClickToShowPopup_Clicked" />
+    <sfPopup:SfPopup x:Name="sfPopup">
+    </sfPopup:SfPopup>
+  </StackLayout>
+</ContentPage.Content>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="4" %}
+private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+{
+  // Shows SfPopup at the bottom of the button and with absolute relative position.
+  sfPopup.ShowRelativeToView(clickToShowPopup, PopupRelativePosition.AlignBottom, 50, 50);
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Displaying a .NET MAUI Popup at Absolute Relative position.](Images/popup-positioning/maui-popup-absolute-relative-positioning.png)
+
+You can pass both negative and positive values as parameters to the `SfPopup.ShowRelativeToView(View, RelativePosition, x-position, y-position)`. The popup will be positioned by considering the relative position as (0, 0) the center point. For example, if you have set the `RelativePosition` as `PopupRelativePosition.AlignBottomRight` and `RelativeView` as a button, bottom right corner of the button will be considered as the 0, 0 point and a negative x-position value will place the popup to the left of that point and a positive x-position value will place the popup to the right of that point. The same applies for y-position also.

```

Comment:
You can pass both negative and positive values as parameters to the `SfPopup.ShowRelativeToView(View, RelativePosition, x and y-position).` The popup will be positioned by considering the relative position as (0, 0) the center point. For example, if you have set the `RelativePosition` as `PopupRelativePosition.AlignBottomRight` and `RelativeView` as a button, bottom right corner of the button will be considered as the 0, 0 point and a negative x-position value will place the popup to the left of that point and a positive x-position value will place the popup to the right of that point. The same applies for y-position.